### PR TITLE
Loki/ES: Add further items to query tracking

### PR DESCRIPTION
**Pull request by svennergr**
**What is this feature?**

This PR adds more items to track in queries executed.

ElasticSearch:
- grafana_version

Loki:
- grafana_version
- time_range_from
- time_range_to
- time_taken
- bytes_processed

### CODE REVIEW

File: public/app/plugins/datasource/loki/tracking.ts [hunk 1]
```diff
@@ -132,18 +132,34 @@ const shouldNotReportBasedOnRefId = (refId: string): boolean => {
...
+): void {
   // We only want to track usage for these specific apps
+  const { app, targets: queries } = request;
+
   if (app === CoreApp.Dashboard || app === CoreApp.PanelViewer) {
     return;
   }
 
+  let totalBytes = 0;
+  for (const frame of response.data as DataFrame[]) {
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/loki/tracking.ts:148]: We should probably not do `as DataFrame` and figure out how can we do this without introducing new assertion.

File: public/app/plugins/datasource/elasticsearch/tracking.ts [hunk 2]
```diff
@@ -121,6 +121,7 @@ export function trackQuery(response: DataQueryResponse, queries: ElasticsearchQu
     }
     reportInteraction('grafana_elasticsearch_query_executed', {
       app,
+      grafana_version: config.buildInfo.version,
       with_lucene_query: query.query ? true : false,
       query_type: getQueryType(query),
       line_limit: getLineLimit(query),
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/elasticsearch/tracking.ts:127]: We could also add here the same props (besides bytes):
```
time_range_from: request?.range?.from?.toISOString(),
time_range_to: request?.range?.to?.toISOString(),
time_taken: Date.now() - startTime.getTime(),
```

**Review by ivanahuckova — COMMENTED** [review]
Nice! Left some feedback bellow.

File: public/app/plugins/datasource/loki/tracking.ts (hunk 1 above)
Comment (svennergr) [reply public/app/plugins/datasource/loki/tracking.ts]: Updated in 6bc1e609400ebb50f860b485307b73ea4359faaf.

File: public/app/plugins/datasource/elasticsearch/tracking.ts (hunk 2 above)
Comment (svennergr) [reply public/app/plugins/datasource/elasticsearch/tracking.ts:132]: Good idea, added in a704feec6e04157a015d545141d5aef841ba443f

**Review by ivanahuckova — APPROVED** [review]
🤩

**Pull request merged**